extensions/wikia/PageShare/PageShareController.class.php

```diff
@@ -38,7 +39,11 @@ private function prepareShareServicesData( $lang ) {
 
 		foreach ( $wgPageShareServices as $service ) {
 			if ( PageShareHelper::isValidShareService( $service, $lang, $isTouchScreen ) ) {
-				$service['icon'] = PageShareHelper::getIcon( $service['name'] );
+				$service['icon'] = DesignSystemHelper::renderSvg(
```

Comment:
`PageShareHelper::getIcon` had a generic share icon fallback for social networks which didn't have a custom icon. I'm afraid that `DesignSystemHelper::renderSvg` would fail to render anything if there's no `'wds-icons-' . $service['name']` in the Design System. And such situation would happen for `wechat` — `wechat.svg` was removed from PageShare icons set, but there's no `wechat.svg` in Design System.